tedium.js

```diff
@@ -203,13 +290,14 @@ Promise.resolve().then(() => {
   const promises = [];
 
   for (const ghRepo of ghRepos) {
-    promises.push(rateLimit(100).then(() => {
+    promises.push(Promise.resolve().then(() => {
       const targetDir = path.join('repos', ghRepo.name);
       let repoPromise;
       if (cleanup.existsSync(targetDir)) {
         repoPromise = nodegit.Repository.open(targetDir);
       } else {
-        repoPromise = nodegit.Clone.clone(ghRepo.clone_url, targetDir, null);
+        repoPromise = rateLimit(100).then(() =>
+            nodegit.Clone.clone(ghRepo.clone_url, targetDir, null));
       }
       return repoPromise.then((repo) => ({
           repo: repo, dir: targetDir, ghRepo: ghRepo}));
```

Comment:
Apologies again for the orthogonal observation: line 304 is missing a semi-colon.